app1/package.json

```diff
@@ -4,8 +4,8 @@
 	"private": true,
 	"devDependencies": {
 		"@types/node": "^15.0.3",
-		"@types/react": "^17.0.5",
-		"@types/react-dom": "^17.0.5",
+		"@types/react": "^18.0.20",
```

Comment:
There is a small typo there. I believe it should be `"@types/react": "^18.0.2"` not `"@types/react": "^18.0.20"`